### ISSUE TITLE
El sistema no se conecta correctamente con la base de datos (Se debe revisar la configuración y conexión de forma local)

### ISSUE THREAD

**Issue by SolgreyDuocUC** [Blacknight3648/digitalfix-frontend#24]
Revisión de la configuración de la cartera de Oracle, configuración de conexión local y pruebas de funcionamiento deben ser ejecutadas

**Comment by SolgreyDuocUC** [comment]
La base de datos se revisó y se arregló la configuración correctamente. Se deja instructivo para poder configurar de ser necesario como respaldo: 

[Conectar Oracle.pdf](https://github.com/user-attachments/files/32311514/Conectar.Oracle.pdf)

<img width="1250" height="649" alt="Image" src="https://github.com/user-attachments/assets/6f655181-d57f-4ff3-9b3b-414b03687589" />
